### ISSUE TITLE
A problem occurred while using typescript

### ISSUE THREAD

**Issue by greenfivebird** [tscanlin/tocbot#290]
I according to the guide for the import
```js
import * as tocbot from 'tocbot';
```
but the page hint me error TypeError: Cannot set properties of undefined (setting 'tocbot')
I don't know how to resolve this problem

**Comment by greenfivebird** [comment]
if use vue3+ts , we should take out 'this' 
```js
typeof global !== 'undefined' ? global : this.window || this.global
```
to
```js
typeof global !== 'undefined' ? global : .window || .global
```

**Comment by tim-scanlin-ck** [comment]
why does `this` cause issues for it?

**Comment by greenfivebird** [comment]
> why does `this` cause issues for it?

Maybe Vue3 problem

**Comment by yannji** [comment]
> I according to the guide for the import
> 
> ```js
> import * as tocbot from 'tocbot';
> ```
> 
> but the page hint me error TypeError: Cannot set properties of undefined (setting 'tocbot') I don't know how to resolve this problem

I had the same problem you had today🙄

**Comment by tscanlin** [comment]
What if we just remove ‘this.’ So it directly references window / global?

**Comment by tscanlin** [comment]
Sorry I missed that you updated your previous comment. Here is a PR to address this:
https://github.com/tscanlin/tocbot/pull/292

**Comment by tscanlin** [comment]
This prerelease has the fix, can one of you help verify it?
`tocbot@4.18.3-0`

**Comment by greenfivebird** [comment]
> This prerelease has the fix, can one of you help verify it? `tocbot@4.18.3-0`
yes ,I verify it, it can be used normally now

**Comment by tscanlin** [comment]
Thanks for verifying @greenfivebird!
The fix is merged and published in `tocbot@4.19.0`
